pkgs/tools/filesystems/fatresize/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, parted, substituteAll, utillinux, ... }:
+
+stdenv.mkDerivation rec {
+  
+  version = "1.0.2";
+  name = "fatresize-${version}";
+
+  src = fetchurl {
+    url = mirror://sourceforge/fatresize/fatresize-1.0.2.tar.bz2;
```

Comment:
You can use `"...${name}.tar.bz2"` here unless you have reason not to (update scripts or somesuch).